src/lib/voiceSearchHelper/index.js

```diff
@@ -0,0 +1,83 @@
+const STATUS_INITIAL = 'initial';
+const STATUS_ASKING_PERMISSION = 'askingPermission';
+const STATUS_WAITING_FOR_AUDIO = 'waitingForAudio';
+const STATUS_WAITING_FOR_VOICE = 'waitingForVoice';
+const STATUS_RECOGNIZING = 'recognizing';
+const STATUS_FINISHED = 'finished';
+
+export default function voiceSearchHelper({ onQueryChange, onStateChange }) {
+  window.SpeechRecognition =
```

Comment:
So we actually need to write to window? I think it be just `const SpeechRecognition = ...` and there is no footprint on the global scope

Reply:
@tkrugg Updated.